polynote-server/src/main/scala/polynote/server/NotebookManager.scala

```diff
@@ -36,7 +36,7 @@ object NotebookManager {
     def listRunning(): RIO[BaseEnv with GlobalEnv, List[String]]
     def status(path: String): RIO[BaseEnv with GlobalEnv, KernelBusyState]
     def create(path: String, maybeContent: Option[String]): RIO[BaseEnv with GlobalEnv, String]
-    def rename(path: String, newPath: String): RIO[BaseEnv with GlobalEnv, String]
+    def copy(path: String, newPath: String, deletePrevious: Boolean): RIO[BaseEnv with GlobalEnv, String]
```

Comment:
TBH I'd rather keep `copy` and `rename` as separate methods, rather than having this boolean flag.

Reply:
Yeah, I was going back and forth about move vs copy a bunch while I was writing this..